src/router.js

```diff
@@ -2,11 +2,22 @@ import tech from "./views/tech.js";
 import article from "./views/article.js";
 import design from "./views/design.js";
 import { page404 } from "./views/404.js";
-import { dummyData } from "./db.js";
 
 const pathToRegex = (path) =>
   new RegExp("^" + path.replace(/\//g, "\\/").replace(/:\w+/g, "(.+)") + "$");
 
+const getArticles = async (subject) => {
+  const dataList = await fetch("/src/db.json")
+    .then((res) => res.json())
+    .then((data) => {
+      if (subject) {
+        return data[subject];
```

Comment:
이 부분도 제가 코멘트 드린 부분 반영하면 이렇게 바뀔 수 있을 것 같아요.

```suggestion
        return data.filter(article => article.type === subject);
```